.github/workflows/ci_test.yml

```diff
@@ -13,7 +13,7 @@ env:
 
 jobs:
   macos-test:
-    name: MacOS CPU Python3
+    name: MacOS CPU Python3 unit-test
```

Comment:
I think `unit-test` isn't needed. It matches better with the other test names if it's removed.